collections/README.md

```diff
@@ -0,0 +1,106 @@
+section of OAFeat, which is in the process of becoming a 'building block in [OGC API - Common - Part 2: Geospatial 
+Data](http://docs.opengeospatial.org/DRAFTS/20-024.html) as the [Collections requirements 
+class](http://docs.opengeospatial.org/DRAFTS/20-024.html#rc_collections-section). Once the Common version is released we will 
+aim to align with it. But it still seems to be in flux.*
+
+## Link Relations
+
+The following Link relations should exist in the Landing Page (root).
+
+| **rel**        | **href**             | **From**       | **Description** |
+| -------------- | -------------------- | -------------- | ---------------- |
+| `root`         | `/`                  | STAC Core      | The root URI |
+| `self`         | `/`                  | OAFeat         | Self reference, same as root URI |
+| `service-desc` | `/api` (recommended) | OAFeat OpenAPI | The OpenAPI service description. Uses the `application/vnd.oai.openapi+json;version=3.0` media type to refer to the OpenAPI 3.0 document that defines the service's API |
+| `child`        | various              | STAC Core      | The child STAC Catalogs & Collections. Provides curated paths to get to STAC Collection and Item objects |
+| `data` | `/collections` | OAFeat | List of Collections |
+
+Additionally, a `service-doc` endpoint is recommended.
+
+| **rel**      | **href** | **From**       | **Description** |
+| ------------ | -------- | -------------- |----------------- |
+| `service-doc`  | `/api.html` (recommended) | OAFeat OpenAPI | An HTML service description.  Uses the `text/html` media type to refer to a human-consumable description of the service |
+
+## Endpoints
+
+| Endpoint                                        | Returns          | Description |
+| ----------------------------------------------- | ---------------- | ----------- |
+| `/collections`                                  | JSON             | Object with a list of Collections contained in the catalog and links |
+| `/collections/{collectionId}`                   | Collection       | Returns single Collection JSON |
+
+STAC API's implementing the Collections class must support HTTP GET operation at `/collections`, with the return JSON document consisting
+of an array of all STAC Collections and an array of Links.
+
+## Example
+
+```json
+{
+	"collections": [
+		{
+			"stac_version": "1.0.0-beta.2",
+			"stac_extensions": [ ],
+			"id": "cool-data",
+			"title": "Cool Data from X Satellite",
+			"description": "A lot of awesome words describing the data",
+			"license": "CC-BY",
+			"extent": {
+				"spatial": {
+					"bbox": [[-135.17, 36.83, -51.24, 62.25]]
+				},
+				"temporal": {
+					"interval": [["2009-01-01T00:00:00Z",null]]
+				}
+			},
+			"links": [
+				{
+					"rel": "self",
+					"type": "application/json",
+					"href": "https://myservice.com/collections/cool-data"
+				},
+			],
+		}
+	],
+	"links": [
```

Comment:
So no other links required: root, parent?

Reply:
I'll improve this example.